### ISSUE TITLE
How the code handle the blank token when decoding logits from a pre-trained acoustic model?

### ISSUE THREAD

**Issue by gelimcgill** [kensho-technologies/pyctcdecode#90]

**Comment by gelimcgill** [comment]
In addition, what about the beam size?
